### ISSUE TITLE
`TypeError: Cannot read properties of undefined (reading 'timeOfDay')`

### ISSUE THREAD

**Issue by drobati** [OrangeJedi/Aerial#132]
**Describe the bug**
"A JavaScript error occurred in the main process" error window, when trying to open app.

`TypeError: Cannot read properties of undefined (reading 'timeOfDay')`

**To Reproduce**
Double-click on the Aerial app.

**Expected behavior**
Not to do this.

**Screenshots**
[Error window](https://i.imgur.com/4JjwuFV.png)
[Windows Version](https://i.imgur.com/5AdkEGY.png)

**Aerial Version**
1.1.0
Windows 11.

**Additional context**
Might be related to location permissions? Not sure why you're not able to read the time of day.

**Comment by drobati** [comment]
I turned off `Prioritize videos for current time of day`, which might be related to that. If I don't update, that probably solved it, and I'll catch the fix later.

**Comment by drobati** [comment]
https://i.imgur.com/sPa5KYm.png

**Comment by drobati** [comment]
I'm noticing a second issue, where when I log in with my fingerprint while on the screensaver, it ends the screensaver and logs in successfully, and within the second locks and starts the screensaver a second time.

After logging in the second time it goes away.

**Comment by OrangeJedi** [comment]
Thanks for letting me know! I took a look at the first error and couldn't get it to reproduce, but I did add some code that should take care of it in the next release. Disabling the `Prioritize videos for current time of day` will definitely stop the error for now, and tbh, not very many videos have day/night tags right now (see #102)

As for the second issue, that is a known problem that I'll hopefully fix soon.
